### DISCUSSION TITLE
Identify PDFs who passed in a previous OCR

### DISCUSSION THREAD

**Discussion by luanmota** [jsvine/pdfplumber#844] [Q&A] [answered]
I faced some PDFs who was digitized and transformed in a "readable" PDF (you can select chars and copy etc). But all this cases the metadata seems to be messed up because the text_extract returns a lot of special characters. Is there any way to identify these files?

**Comment by jsvine** [comment, marked as answer]
Unfortunately, I'm not aware of a foolproof way of identifying such PDFs. But you might be able to devise an heuristic that works for you, based on some combination of `pdf.metadata` and the presence of full-page images. Haven't tried this myself, though.

**Reply by luanmota** [reply]
the ideia to use pdf.metadata helped a lot! thanks again @jsvine
